src/test/java/com/wikia/webdriver/testcases/communityheadertests/CommunityHeaderTests.java

```diff
@@ -50,8 +51,8 @@ public void testLoggedInWikiButtons() {
     communityHeader.clickWikiActivity();
     Assert.assertTrue(driver.getCurrentUrl().contains("Special:WikiActivity"));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Special:WikiActivity" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)